src/csharp/Grpc.IntegrationTesting.Client/project.json

```diff
@@ -13,8 +13,10 @@
           "includeFiles": [ "../Grpc.Core/Version.cs" ]
         },
         "copyToOutput": {
-          "include": "data/*",
```

Comment:
I think it makes sense how this would not work with dotnet cli, as it looks to bet trying to copy the data/ directory from the separate project, Grpc.IntegrationTesting, but it seems strange how this works with xbuild.